Makefile

```diff
@@ -138,6 +138,21 @@ endif
 LD_FLAGS        :=
 EXTRA_LD_FLAGS  :=
 
+#
+# Setup locale
+#
+LOCALE_LIST := $(sort $(notdir $(patsubst %/,%,$(dir $(wildcard $(SRC_DIR)/locales/*/bf_locale.xml)))))
+ifeq ($(LOCALE),)
+LOCALE := en
+endif
+ifeq ($(filter $(LOCALE),$(LOCALE_LIST)),)
+    $(error LOCALE $(LOCALE) must be one of >$(LOCALE_LIST)<)
+endif
+INCLUDE_DIRS += $(INCLUDE_DIRS) $(SRC_DIR)/locales/$(LOCALE)
```

Comment:
Is it really necessary to pollute include search path? It is already quite overused in BF

Reply:
I think so, this was a suggestion from @blckmn

Reply:
Using `-DLANG=en` should do. You can use it in `#include`:
```
#define LOCALE_FILE(lang) locales/lang/bf_locale.h
#include LOCALE_FILE